tools/gen-intrinsics-specs.py

```diff
@@ -41,42 +41,67 @@ def quote_literal(val):
 __SECTION_KEYWORDS = [__INTRINSIC_TABLE_KEYWORD, __SECTION_TEXT_KEYWORD]
 
 
-def rst_literal_quote(mapping):
+def literal_quote(mapping, table_format):
     r"""
-    >>> rst_literal_quote('a; b')
+    >>> literal_quote('a; b', 'grid')
     ' ::\n\n    a \n    b \n\n'
 
-    >>> rst_literal_quote('a b')
+    >>> literal_quote('a b', 'grid')
     ' ::\n\n    a b \n\n'
 
-    >>> rst_literal_quote('')
+    >>> literal_quote('', 'grid')
+    ''
+    >>> literal_quote('a; b', 'github')
+    '<br><br>    a <br>    b <br><br>'
+
+    >>> literal_quote('a b', 'github')
+    '<br><br>    a b <br><br>'
```

Comment:
In this case, instead, because there is no `;` in the input, the string is not split. So all we need to see here  is the `s wrapped string, as follows:
```
`a b`
```